inst/ad_advs.R

```diff
@@ -0,0 +1,260 @@
+# Name: ADVS
+#
+# Label: Vital Signs Analysis dataset
+#
+# Input: adsl, vs
+
+
+# Attach/load required packages ----
+library(admiral)
+library(admiralmetabolic)
+library(tibble)
+library(dplyr)
+library(stringr)
+
+
+# Define look-up tables ----
+# See function documentation for `derive_vars_merged_lookup()` and
+# `derive_vars_cat()` for information of how look-up tables can be used:
+# (https://pharmaverse.github.io/admiral/reference/derive_vars_merged_lookup.html)
+# (https://pharmaverse.github.io/admiral/reference/derive_vars_cat.html)
+
+# Parameter look-up
+param_lookup <- tribble(
```

Comment:
we agreed that for PARAMN in range 1-5 PARCAT1 = "Anthropometric measurements"

Reply:
Yeah, I could not remember what we agreed. We should also update the vignette.

Reply:
It is changed. I will update the issue for the vignette before resolving the conversation.